src/audioWorklet/index.js

```diff
@@ -9,12 +9,14 @@ let initializedAudioWorklets = false;
 
 function loadAudioWorkletModules() {
   return Promise.all(
-    moduleSources.map(function (moduleSrc) {
+    moduleSources.map(function(moduleSrc) {
       const blob = new Blob([moduleSrc], { type: 'application/javascript' });
       const objectURL = URL.createObjectURL(blob);
-      return ac.audioWorklet.addModule(objectURL);
-    })
-  );
+      return ac.audioWorklet
+      .addModule(objectURL)
+      // in "p5 instance mode," the module may already be registered
+      .catch(() => Promise.resolve());
```

Comment:
Just a suggestion, it might be useful for development checking what the error is here, so e.g. if someone's making a change and has a syntax error, they can still see that error here.